object-manipulation-3/main.js

```diff
@@ -0,0 +1,137 @@
+console.log('Lodash is loaded:', typeof _ !== 'undefined');
+
+var players = [
+  {
+    name: 'Ron',
+    hand: []
+  },
+  {
+    name: 'Leslie',
+    hand: []
+  },
+  {
+    name: 'Ben',
+    hand: []
+  },
+  {
+    name: 'April',
+    hand: []
+  }
+];
+
+var cardDeck = [
+  { rank: 'Ace', suit: 'clubs' },
+  { rank: 2, suit: 'clubs' },
+  { rank: 3, suit: 'clubs' },
+  { rank: 4, suit: 'clubs' },
+  { rank: 5, suit: 'clubs' },
+  { rank: 6, suit: 'clubs' },
+  { rank: 7, suit: 'clubs' },
+  { rank: 8, suit: 'clubs' },
+  { rank: 9, suit: 'clubs' },
+  { rank: 10, suit: 'clubs' },
+  { rank: 'Jack', suit: 'clubs' },
+  { rank: 'Queen', suit: 'clubs' },
+  { rank: 'King', suit: 'clubs' },
+  { rank: 'Ace', suit: 'diamonds' },
+  { rank: 2, suit: 'diamonds' },
+  { rank: 3, suit: 'diamonds' },
+  { rank: 4, suit: 'diamonds' },
+  { rank: 5, suit: 'diamonds' },
+  { rank: 6, suit: 'diamonds' },
+  { rank: 7, suit: 'diamonds' },
+  { rank: 8, suit: 'diamonds' },
```

Comment:
Try generating your deck with a couple of loops and an array of suits and array of ranks.